### PR TITLE
Generated protofiles based on v0.7.0

### PR DESCRIPTION
**Pull request by MadVikingGod**
This is the generation of v0.7.0 of the Opentelemetry-proto repository.

commands to generated this:
```
cd opentelemetry-proto
git checkout v0.7.0
cd ..
make
```

### CODE REVIEW

**Review by MrAlias — APPROVED** [review]

File: otlp/go.mod [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module go.opentelemetry.io/proto/otlp
```
Comment (tigrannajaryan) [inline otlp/go.mod:1]: Do we use vanity url for this repo? I don't see `go.opentelemetry.io/proto` enabled.

**Comment by tigrannajaryan** [comment]
> commands to generated this:
> 
> ```
> cd opentelemetry-proto
> git checkout v0.7.0
> cd ..
> make
> ```

Can we add these as a github action that verifies that the result of the generation matches the content in the commit?

File: otlp/go.mod (hunk 1 above)
Comment (MadVikingGod) [reply otlp/go.mod:1]: [Not yet](https://github.com/open-telemetry/opentelemetry-go-vanityurls/pull/11).  I was waiting until there is usable logic in this repository to enable that URL.

**Comment by MadVikingGod** [comment]
Agreed, that is something that was planned.  Tracked in #6

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**